frontend/server/server.js

```diff
@@ -1,11 +1,22 @@
 import http from 'http';
+import path from 'path';
 import express from 'express';
 
 let app = express();
+app.set('views', path.join(__dirname, 'views'));
+app.set('view engine', 'ejs');
 app.use(express.static(__dirname + '/../dist'));
 
 let server = http.createServer(app);
 
+app.get('*', (req, res) => {
+  res.render('index.ejs');
+});
+
+// app.get('*', (req, res) => {
+//   res.sendFile(path.join(__dirname, '../dist/index.html'));
+// });
```

Comment:
Tested this method (commented out above) as an alternative to res.render('index.ejs'). It works, but unless you have particular objections, I would prefer rendering the template on the server.